csm/core/services/rgw/s3/utils.py

```diff
@@ -30,7 +30,7 @@ def get_rgw_connection_config():
         """Creates a configuration for RGW connection."""
         rgw_connection_config = RgwConnectionConfig()
         rgw_endpoint = Conf.get(
-            const.CSM_GLOBAL_INDEX, 'RGW>s3>endpoints[0]')
```

Comment:
If you are not using RGW>s3>endpoints anywhere, please remove its instances in the code.

Reply:
Not using it.. already removed from code